Tallies/TallyResponses/Tests/macroResponse_test.f90

```diff
@@ -102,13 +111,15 @@ subroutine testGettingResponse(this)
     real(defReal), parameter :: TOL = 1.0E-9
 
     p % type = P_NEUTRON
+    p % isMG = .false.
```

Comment:
Why did this have to be added? Perhaps I missed something. Is there a reason why energy deposition can't be included with MG? Perhaps for a second PR, but it would be nice to have a kappa value in the MG input that could be used.